pkg/auth/fleetshard_authz.go

```diff
@@ -40,6 +42,8 @@ func (c *FleetShardAuthZConfig) AddFlags(fs *pflag.FlagSet) {
 		"Fleetshard authZ middleware configuration file containing a list of allowed org IDs")
 	fs.BoolVar(&c.Enabled, "enable-fleetshard-authz", c.Enabled, "Enable fleetshard authZ "+
 		"via the list of allowed org IDs")
+	fs.StringVar(&c.AllowedSubject, "fleetshard-authz-subject", c.AllowedSubject,
+		"Fleetshard authZ middleware allowed subject")
```

Comment:
nit: maybe to make it more clear to folks reading through things:
```suggestion
		"Fleetshard authZ middleware allowed subject claim of the token")
```

Reply:
Obsolete, moved to the config file